tests/python/rest_api/test_labels.py

```diff
@@ -594,11 +613,24 @@ def _get_patch_data(
             payload = deepcopy(overrides)
 
             if overrides.get("attributes"):
-                payload["attributes"] = (original_data.get("attributes") or []) + overrides[
-                    "attributes"
-                ]
+                filtered_attributes = self._attributes_with_id(deepcopy(original_data.get("attributes", [])))
```

Comment:
Why do we need to use `_attributes_with_id` here? Since `original_data` represents a created label taken from assets, then if a label contains attributes -> each attribute has `id`.

Reply:
I removed this extra step.